### PR TITLE
Updates to view query parameters for 86965

### PR DESCRIPTION
**Pull request by lbboe**
Made recommended changes to Accessing a stale view topic based on case and Rob's comments.

### CODE REVIEW

File: docs2017/src/api/using_views.md [hunk 1]
```diff
@@ -183,74 +183,64 @@ a stale view might not return the most recent information.
...
+
+When the default `stable=false` value applies,
+and you use either `update=false` or `update=lazy`,
+indexes between shard replicas are no longer synchronized.
+The results are different based on which replica responds first. 
+
+When you use a stale view, the results return the existing version of the data in the view index without waiting for an update. 
+The results can be different from different nodes in the cluster.
+
+>   **Note**: Cloudant automatically and acynchronously keeps views in sync. If you use the `stable` or
```
Comment (rnewson) [inline docs2017/src/api/using_views.md]: is this note needed given the new beginning?

Comment (lbboe) [inline docs2017/src/api/using_views.md]: I thought about that too. I don't think so. I'll remove it.

**Comment by lbboe** [comment]
Rob, Adrian reminded me that we are not to use "recommend," "fastest," and "best" in documentation. I've rewritten the text to remove those. There is also a confusing sentence, "Do not use `stale=ok` or the new `stable` and `update` parameters by default." Can you give me more description for this one, please? Thanks.

**Comment by rnewson** [comment]
It's a nice improvement to a complicated area of the API, thanks!

**Comment by rnewson** [comment]
You can strike that "Do not use stale=ok or the new stable and update parameters by default." sentence, agree it's confusing.

**Pull request merged**